include/boost/geometry/algorithms/detail/distance/multipoint_to_geometry.hpp

```diff
@@ -126,14 +126,11 @@ class multipoint_to_areal
                                     Areal const& areal,
                                     Strategies const& strategies)
     {
-        not_covered_by_areal predicate(areal, strategies);
+        covered_by_areal predicate(areal, strategies);
 
-        if (check_iterator_range
-                <
-                    not_covered_by_areal, false
-                >::apply(boost::begin(multipoint),
-                         boost::end(multipoint),
-                         predicate))
+        if (boost::end(multipoint) == std::find_if(boost::begin(multipoint),
+                                                   boost::end(multipoint),
+                                                   predicate))
```

Comment:
For empty multipoint the result was `false` here before, now it would be `true`. There is no need to call whatever is below for empty geometry while calculating distance.